test-files/ui/features/step_definitions/custom_steps.rb

```diff
@@ -334,7 +334,7 @@
 end
 
 When(/^I fill "([^"]*)" with "([^"]*)"$/) do |el, value|
-  find('#' + el).set(value)
+  find(el).set(value)
```

Comment:
Thought I had checked for this when I changed it but I guess not. I updated the `clip.feature` test where this step was also being used.